_pytest/logging.py

```diff
@@ -364,6 +364,13 @@ def __init__(self, config):
         # initialized during pytest_runtestloop
         self.log_cli_handler = None
 
+    def _log_cli_enabled(self):
+        """Return True if log_cli should be considered enabled, either explicitly
+        or because --log-cli-level was given in the command-line.
+        """
+        return self._config.getoption('--log-cli-level') is not None or \
+               self._config.getini('log_cli')
```

Comment:
Thx for introducing a method for this! This makes it a bit easier to understand.